### ISSUE TITLE
Expected Center Loss

### ISSUE THREAD

**Issue by KiryanovKD** [tensorflow/addons#643]
**System information**

TensorFlow version (you are using): 2 rc
Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Expected Center Loss can help in learning discriminative features in seq training without framewising

**Will this change the current api? How?**
No

**Who will benefit with this feature?**
Anyone who wants to use this loss function in ASR or OCR

**Any Other info.**
https://arxiv.org/pdf/1905.06907.pdf

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
